### PR TITLE
feat(pci.training): datatr-155 - jobs privacy settings

### PR DESCRIPTION
**Pull request by jperchoc**
DATATR-155

| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #DATATR-155
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Description

Add privacy settings for training jobs.

### :flags: Translations
- [ ] TRANS-48614

### CODE REVIEW

**Review by ClaveauAlex — DISMISSED** [review]

**Review by Dimfacion — DISMISSED** [review]

File: packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.html [hunk 1]
```diff
@@ -139,6 +139,50 @@ <h2>
...
+                        <span
+                            data-translate="pci_projects_project_training_jobs_submit_select_privacy_settings_item_restricted_description"
+                        ></span>
+                    </oui-select-picker-section>
+                </oui-select-picker>
+
+                <!--Settings public-->
+                <oui-select-picker
+                    class="d-inline-block col-md-6 col-xl-3 my-3"
+                    data-name="rivacyNetwork"
```
Comment (ghyenne) [inline packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.html:166]:
```suggestion
                    data-name="privacyNetwork"
```

Comment (ghyenne) [reply packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.html:166]: But already exist above.

**Comment by ghyenne** [comment]
I see that "Breaking change is mentioned in relevant commits" is checked. What is the breacking change ?

File: packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.html (hunk 1 above)
Comment (jperchoc) [reply packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.html]: Changed here: https://github.com/ovh/manager/pull/8578/commits/22e7ec496e96b57be275845ff1602d09256fc063

**Review by ClaveauAlex — DISMISSED** [review]

File: packages/manager/modules/pci/src/projects/project/training/jobs/info/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -29,5 +29,9 @@
   "pci_projects_project_training_jobs_info_title_6": "Infos",
   "pci_projects_project_training_jobs_info_title_actions": "Actions",
   "pci_projects_project_training_job_logs_text_pre_running": "Le job est encore en attente de démarrage… L'onglet se rafraîchira automatiquement une fois que le job aura commencé.",
-  "pci_projects_project_training_jobs_resubmit": "Relancer"
+  "pci_projects_project_training_jobs_resubmit": "Relancer",
+
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/training/jobs/info/translations/Messages_fr_FR.json:33]: **nitpick:** Can you remove this empty line?
```suggestion
```

Comment (jperchoc) [reply packages/manager/modules/pci/src/projects/project/training/jobs/info/translations/Messages_fr_FR.json]: Sure, done here: https://github.com/ovh/manager/pull/8578/commits/5dba8a2bed58e82a1c550049e847285f3031e864

**Review by ClaveauAlex — APPROVED** [review]

**Review by anooparveti — DISMISSED** [review]

**Review by sachinrameshn — APPROVED** [review]

**Pull request merged**